OpenRA.Mods.Common/Activities/Transform.cs

```diff
@@ -85,6 +85,15 @@ void DoTransform(Actor self)
 				if (self.IsDead || self.WillDispose)
 					return;
 
+				// Prevent deployment in bogus locations
+				if (transforms != null && !transforms.CanDeploy())
+				{
+					if (!SkipMakeAnims && makeAnimation != null)
+						makeAnimation.Forward(self, () => { IsInterruptible = true; Cancel(self, true); });
```

Comment:
```suggestion
					if (!SkipMakeAnims && makeAnimation != null)
						makeAnimation.Forward(self, () => { IsInterruptible = true; Cancel(self, true); });
					else
						Cancel(self, true);
```
?

Reply:
These are the same checks that exist in Tick so there'd be nothing to cancel when they are false. Though I guess I can add it for completeness